### PR TITLE
Fixed deprecated function calls

### PR DESCRIPTION
**Pull request by jkimbrel**

### CODE REVIEW

File: src/Mailer/UsersMailer.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected function validation(EntityInterface $user)
             ->setTo($user['email'])
              ->setSubject($firstName . $subject)
              ->setViewVars($user->toArray())
-             ->setTemplate('CakeDC/Users.validation');
+             ->viewBuilder()->setTemplate('CakeDC/Users.validation');
```
Comment (rochamarcelo) [inline src/Mailer/UsersMailer.php]: Please update unit test  (UsersMailerTest)

File: src/Mailer/UsersMailer.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ protected function resetPassword(EntityInterface $user)
             ->setTo($user['email'])
             ->setSubject($subject)
             ->setViewVars($user->toArray())
-            ->setTemplate('CakeDC/Users.resetPassword');
+            ->viewBuilder()->setTemplate('CakeDC/Users.resetPassword');
```
Comment (rochamarcelo) [inline src/Mailer/UsersMailer.php]: Please update unit test  (UsersMailerTest)

File: src/Mailer/UsersMailer.php [hunk 3]
```diff
@@ -76,6 +76,6 @@ protected function socialAccountValidation(EntityInterface $user, EntityInterfac
             ->setTo($user['email'])
             ->setSubject($subject)
             ->setViewVars(compact('user', 'socialAccount'))
-            ->setTemplate('CakeDC/Users.socialAccountValidation');
+            ->viewBuilder()->setTemplate('CakeDC/Users.socialAccountValidation');
```
Comment (rochamarcelo) [inline src/Mailer/UsersMailer.php]: Please update unit test  (UsersMailerTest)

**Comment by challgren** [comment]
This could be closed because 8.next has all the 3.7 deprecations fixed

**Pull request closed without merging**